code/modules/vehicles/interior/interactable/seats.dm

```diff
@@ -448,6 +448,57 @@
 		if(EXPLOSION_THRESHOLD_MEDIUM to INFINITY)
 			break_seat()
 
+
+/obj/structure/bed/chair/vehicle/dropship_cockpit
+	name = "cockpit seat"
+	desc = "A sturdy metal chair with a brace that lowers over your body. Holds you in place during high altitude drops."
+	icon_state = "vehicle_seat"
+	can_rotate = FALSE
+
+/obj/structure/bed/chair/vehicle/dropship_cockpit/afterbuckle(mob/M)
```

Comment:
single letter var, why override this proc? I'm not seeing what this does that the parent doesn't on a quick read.

Reply:
Keeps the buckled mob undense so that you can walk through them, for the Tripoli/Midway 

![image](https://github.com/user-attachments/assets/0c2606d6-b151-4cf1-8c4d-139a1908fc84)